### PR TITLE
Users/zhilmil/bugfix

### PR DESCRIPTION
**Pull request by v-zhgup**
## Description
Changes to remove hard coded value of storage name and printing the permission warning message in PowerShell output log file
 </br>

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK/Framework/Core/SVT/Services/KeyVault.ps1 [hunk 1]
```diff
@@ -74,78 +104,86 @@ class KeyVault: SVTBase
...
-		}
+			
 		return $this.AllEnabledKeys;
 	}
 
 	hidden [PSObject[]] FetchAllEnabledSecretsWithVersions([ControlResult] $controlResult)
 	{
-		if($null -eq $this.AllEnabledSecrets)
+
+		if($this.HasFetchKeysPermissions -eq $true)
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Core/SVT/Services/KeyVault.ps1]: We need to set has required access to false for control...
$controlResult.CurrentSessionContext.Permissions.HasRequiredAccess = $false;

Comment (v-zhgup) [reply src/AzSK/Framework/Core/SVT/Services/KeyVault.ps1]: Done

File: src/AzSK/Framework/Core/SVT/Services/KeyVault.ps1 [hunk 2]
```diff
@@ -31,6 +34,33 @@ class KeyVault: SVTBase
...
+		$currentContext=[Helpers]::GetCurrentRMContext();
+		$CurrentContextId=$currentContext.Account.Id;
+		$CurrentContextObjectId=$null
+		if($currentContext.Account.Type -eq 'User')
+		{
+			$CurrentContextObjectId=Get-AzureRmADUser -UserPrincipalName $CurrentContextId|Select-Object -Property Id
+		}
+		elseif($currentContext.Account.Type -eq 'ServicePrincipal')
+		{
+			$CurrentContextObjectId=Get-AzureRmADServicePrincipal -ServicePrincipalName $CurrentContextId|Select-Object -Property Id
```
Comment (sbyna-msft) [inline src/AzSK/Framework/Core/SVT/Services/KeyVault.ps1]: Try with SPN login

Comment (v-zhgup) [reply src/AzSK/Framework/Core/SVT/Services/KeyVault.ps1]: Added try catch for exception handling.

**Pull request merged**